package.json

```diff
@@ -7,7 +7,14 @@
   "author": "PermissionError",
   "license": "MIT",
   "private": false,
+  "scripts": {
+    "pretest": "eslint --ignore-path .gitignore ."
+  },
   "dependencies": {
     "express": "^4.17.1"
+  },
+  "devDependencies": {
+    "eslint": "^6.8.0",
+    "eslint-config-strongloop": "^2.1.0"
```

Comment:
why you need this?

Reply:
React-4 specifically asks us to use ESLint and those were installed according to the blog post in the task instructions...???

Reply:
ES-Lint is enough. Any reasons for this dependency? You should know the reason behind this dependency before adding that to your project.

Reply:
Paraphrasing the blog post: StrongLoop is unremarkable and similar to the syntax used in many open-source Node.js projects.

Also: I thought you meant why ESLint was added. Due to the nature of GitHub's UI, I can't see which line you specifically commented for, my apologies.

Reply:
Have you seen this : https://www.npmtrends.com/eslint-config-strongloop-vs-eslint-config-recommended-vs-eslint-config-google
By the way, Keep it for now.